### PR TITLE
Feature/attribute services

### PR DESCRIPTION
**Pull request by LindaNieves23**
AttributeServices

Component for Mobile and desktop created including the file:

HBS:
 serviceAttribute.hbs
 serviceAttributesContainer.hbs

CSS:
 _serviceAttributesContainer.scss

Test Page:
http://localhost:8080/AttributesPage.html

Link Trello:
https://trello.com/c/ajU7IArq

### CODE REVIEW

**Review by jrmahechad — APPROVED** [review]

File: templates/serviceAttribute.hbs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<div class='container__attribute-section--title container__attribute-section--box'>{{title}}
```
Comment (jrmahechad) [inline templates/serviceAttribute.hbs]: This should be a p tag

File: templates/serviceAttribute.hbs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<div class='container__attribute-section--title container__attribute-section--box'>{{title}}
+
+</div>
+
+<div class='container__attribute-section--value container__attribute-section--box'>{{values}}
```
Comment (jrmahechad) [inline templates/serviceAttribute.hbs]: This should be a p tag

**Review by jrmahechad — CHANGES_REQUESTED** [review]
@LindaNieves23 great job. I added a small comment

File: templates/serviceAttributesContainer.hbs [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+      {{#each attributes}}
+        {{{module this}}}
+      {{/each}}
+    </div>
+
+    <div class='container__line-divider'></div>
+
+  </div>
+
+  <a class='link-date'>{{{module bookNowButton}}}</a>
```
Comment (jsebax) [inline templates/serviceAttributesContainer.hbs]: If you put the `module bookNowButton` inside an `<a>` tag, you'll have two anchor links. Remove the `<a>` tag around and just leave the `{{{module bookNowButton}}}` code, and use the `.link` class that the `bookNowButton` has to specify your button styles.

**Review by jrmahechad — APPROVED** [review]

**Review by jrmahechad — APPROVED** [review]

**Pull request merged**